packages/components/Avatar/src/index.ts

```diff
@@ -1,3 +1,4 @@
 export * from './Avatar';
 export * from './Avatar.types';
 export { useAvatar } from './useAvatar';
+export { getHashCodeWeb } from './getHashCode';
```

Comment:
Does this mean we always export `getHashCodeWeb` to all customers? Is that something we want to do?

Reply:
We used it for win32 because we were following the Web implementation. I'm not sure if each platform should have its own getHashCode algorithm. @rurikoaraki , what do you think?

Reply:
Oh, I see that algorithm is changed back :)

Reply:
So do we not want to export this?

Reply:
Sorry for misleading comment. How you implemented it, it's good. Each platform will have it's own getHashCode algorithm. @rurikoaraki , correct me if I'm wrong

Reply:
Ohh ok awesome, thank you!